app/components/autocomplete/autocomplete_section_header.js

```diff
@@ -12,11 +12,12 @@ export default class AutocompleteSectionHeader extends PureComponent {
     static propTypes = {
         defaultMessage: PropTypes.string.isRequired,
         id: PropTypes.string.isRequired,
+        loading: PropTypes.bool,
```

Comment:
Any reason not to make this required?

Reply:
Just personal preference really.
- Explicitly passing `loading={false}` to me means the same as not passing it at all.
- It adds an unneeded prop/line when using the component if the prop is required.

Plus there's the argument that the component doesn't necessarily depend on the prop to render successfully.